wire-library/wire-grpc-server/build.gradle

```diff
@@ -0,0 +1,25 @@
+apply plugin: 'java-library'
+apply plugin: 'org.jetbrains.kotlin.jvm'
+
+jar {
+    manifest {
+        attributes('Automatic-Module-Name': 'wire-grpc-server')
+    }
+}
+
+dependencies {
+    implementation project(':wire-runtime')
+    // io.grpc.stub relies on guava-android. This module relies on a -jre version of guava.
+    implementation(deps.grpc.stub) {
+        exclude(group: 'com.google.guava', module: 'guava')
+    }
+    implementation ("com.google.guava:guava:21.0")
```

Comment:
21 ?

Reply:
I've kept this at 21 given it's the oldest possible one that supports Monitor, and the rest of Wire is at minimum 20.

Reply:
We should bump it to something current! But no need for doing that in this PR